wayback-webapp/src/main/webapp/WEB-INF/ComplexAccessPoint.xml

```diff
@@ -161,7 +166,11 @@
         </property>
       </bean>
     </property>
-
+    <property name="trustedProxies">
+      <list>
+        <value>140.147.210.0/24</value>
+      </list>
+    </property>
```

Comment:
Hey @chasehd , I think the `<property name="trustedProxies">` block should be moved up to line 167 in this example. Also, I think as far as making a more helpful/working example, what do you think about adding to the `netsecureaccesspoint`:
```
    <property name="replayPrefix" value="${wayback.url.prefix}/netsecure/" />
    <property name="queryPrefix" value="${wayback.url.prefix}/netsecure/" />
    <property name="staticPrefix" value="${wayback.url.prefix}/netsecure/" />
```
I think with out it, if this example is followed by someone, a user might start a search on the `/netsecure` path, but then they would be dumped back to the path of the `standardaccesspoint`.
Also we could change `<property name="accessPointPath" value="http://localhost:8080/netsecure/"/>` to use `${wayback.url.prefix}`.